leptos/src/show.rs

```diff
@@ -51,8 +51,9 @@ where
 {
     let memoized_when = create_memo(move |_| when());
 
-    move || match memoized_when.get() {
-        true => children().into_view(),
-        false => fallback().into_view(),
+    if memoized_when.get() {
```

Comment:
dropping the closure here is a behavior change that introduces a bug — whether `children` or `fallback` is shown will not be reactive any more